backend/src/interface/controller/bookController.ts

```diff
@@ -106,8 +106,7 @@ export default class BookController {
 
     const pageNumber = page ? parseInt(page as string, 10) : undefined;
     const perPageNumber = perPage ? parseInt(perPage as string, 10) : undefined;
-
-    const [books, error] = await wrapError(
+    const [book, error] = await wrapError(
```

Comment:
Aquí si debe ser "books":
```suggestion
    const [books, error] = await wrapError(
```